dotCMS/src/main/java/com/dotmarketing/business/APILocator.java

```diff
@@ -337,7 +337,10 @@ public static RelationshipAPI getRelationshipAPI(){
 	public static com.dotmarketing.portlets.structure.business.FieldAPI getFieldAPI(){
 		return (com.dotmarketing.portlets.structure.business.FieldAPI)getInstance(APIIndex.FIELD_API);
 	}
-
+	
+    public static com.dotcms.contenttype.business.FieldAPI  getFieldAPI2(){

```

Comment:
Well, Jose,  your suggestion was so good that it was already implemented like that.  Turns out we don't even need the getFieldAPI2  method.